Tests/CarthageKitTests/DependencySpec.swift

```diff
@@ -1,3 +1,4 @@
+@testable import CarthageKit
```

Comment:
one of the two imports is redundant, see next line

Reply:
Remove initial import